libpretixsync/src/main/java/eu/pretix/libpretixsync/models/TaxRule.kt

```diff
@@ -7,4 +7,5 @@ data class TaxRule(
     val serverId: Long,
     val rate: BigDecimal = BigDecimal("0.00"),
     val includesTax: Boolean = false,
+    val code: String?,
```

Comment:
This could default to `null` since the JSON parsing does the same. Makes creating models in tests more convenient